Author: gregor herrmann <gregoa@debian.org>
Description: we don't want to run `perl Changes.PL' since
             it would need a git repo

--- a/Makefile.PL
+++ b/Makefile.PL
@@ -8,4 +8,5 @@
               dist          => { COMPRESS => "gzip", SUFFIX => "gz" },
               clean         => { FILES => '*.bak *.old mibs/*.dump *.pdf' .
                                           'lib/*/*~ lib/*/*/*~' },
+              PL_FILES      => {},
 );
